### ISSUE TITLE
Simple string fontSize values not handled correctly

### ISSUE THREAD

**Issue by fabianeichinger** [semencov/tailwindcss-font-inter#13]
We've some custom font sizes configured in our Tailwind theme, like so:
```
{
  theme: {
    extend: {
      fontSize: {
        'xxs': '.75rem',
        'xs': '.8125rem'
      }
    }
  }
}
```

Version 2.0.2 of this plugin generates broken CSS rules for this configuration
![image](https://user-images.githubusercontent.com/4332237/112000408-42be9c80-8b1e-11eb-9a79-7ebde01296cf.png)
(The duplicate rules are probably due to an unrelated issue, maybe with tailwind-jit)

The offending line seems to be https://github.com/semencov/tailwindcss-font-inter/blob/e7ef43b44b10469342b34a304be2c1f3331afaa9/src/index.js#L88
- `fontSize` is the String `".8125rem"`
- `isArrayLike` returns `true` for a String
- `size` is only the first character of the fontSize value, i.e. `"."`

## Workaround

We've worked around this for now by replacing the string values with arrays
```
{
  theme: {
    extend: {
      fontSize: {
        'xxs': ['.75rem', {}],
        'xs': ['.8125rem', {}]
      }
    }
  }
}
```
Result:
![image](https://user-images.githubusercontent.com/4332237/112001073-e314c100-8b1e-11eb-8db4-862132bfc7fa.png)

**Comment by semencov** [comment]
@fabianeichinger, thank you for the report. Could you downgrade the plugin to v2.0.1 and test it? I'll fix the issue on the weekend.

Also, I will be appreciated if you could comment in #12 if you have any investigation about the tailwind-jit issue.

**Comment by fabianeichinger** [comment]
@semencov Thank you for taking on this issue. I'm looking forward to remove the workaround!
Our dependencies are `{ "tailwindcss": "2.0.4", "tailwind-jit": "0.1.5" }`. With v2.0.1 of your plugin we're running into https://github.com/semencov/tailwindcss-font-inter/issues/8 for the font sizes that are part of Tailwind's default theme (and also our custom font sizes when my array workaround is applied).

I have not yet investigated the duplicate rules though, sorry!

**Comment by semencov** [comment]
Ok, got it. Will try to release the fix within the holidays.

**Comment by semencov** [comment]
Hi @fabianeichinger, I've pushed a quick fix. Could you please update the plugin to v2.0.3 and test it?

**Comment by fabianeichinger** [comment]
@semencov Thanks, seems to be fixed in 2.0.3
